src/bitpacking_cmp.rs

```diff
@@ -0,0 +1,128 @@
+use crate::FastLanes;
+use crate::{BitPackWidth, BitPacking, SupportedBitPackWidth};
+
+pub trait BitPackingCompare: BitPacking {
```

Comment:
I don't really understand what the three functions are for now?

Reply:
I understand what you meant and resolved this